integration_tests/src/main/python/cache_test.py

```diff
@@ -20,12 +20,16 @@
 from spark_session import with_cpu_session, with_gpu_session
 from join_test import create_df
 from marks import incompat, allow_non_gpu, ignore_order
-from parquet_write_test import parquet_decimal_struct_gen
 
-enableVectorizedConf = [{"spark.sql.inMemoryColumnarStorage.enableVectorizedReader" : "true"},
-                        {"spark.sql.inMemoryColumnarStorage.enableVectorizedReader" : "false"}]
+enableVectorizedConfs = [{"spark.sql.inMemoryColumnarStorage.enableVectorizedReader": "true"},
+                        {"spark.sql.inMemoryColumnarStorage.enableVectorizedReader": "false"}]
+allowNegativeScaleDec = {"spark.sql.legacy.allowNegativeScaleOfDecimal": "true"}
```

Comment:
This already exists.

https://github.com/NVIDIA/spark-rapids/blob/231f8e30bc4cd117a891888147e637985b3da773/integration_tests/src/main/python/data_gen.py#L876

datagen.py is probably not the most ideal place for it, but it works.